lib/src/modules/apply.cc

```diff
@@ -265,7 +265,8 @@ exit exit_code
 
         const auto plugin_cache = module_cache_dir_->createCacheDir(plugin_cache_name);
         params.set<std::string>("plugin_cache", plugin_cache.string());
-        params.set<std::vector<std::string>>("master_uris", master_uris_);
+        params.set<std::vector<std::string>>("master_uris", primary_uris_);
```

Comment:
I dont think we need this anymore.

Reply:
I wasn't sure if removing this would cause some issue with backwards compatibility since I don't fully understand how it gets used, but I can try dropping it

Reply:
you pass whatever you want the arg name through and its used https://github.com/puppetlabs/pxp-agent/pull/830/commits/b4ee333f111dbbeb7ae72dfe42cfaeca15299d3a#diff-47d2fffa351567b475e3243c3a80d4b751e29a9057e424a3709cebee6ebfc47fR78 so just get rid of master_uris in both places.

Reply:
Ah yep that's easy. Thanks

Reply:
Done!